src/routing/InitialisedRouter.tsx

```diff
@@ -25,15 +22,11 @@ import { NewProbe } from 'page/NewProbe';
 import { CheckNotFound } from 'page/NotFound/CheckNotFound';
 import { PluginPageNotFound } from 'page/NotFound/NotFound';
 import { Probes } from 'page/Probes';
-import { ProbesWelcomePage } from 'page/ProbesWelcomePage';
 import { SceneHomepage } from 'page/SceneHomepage';
-import { UnprovisionedSetup } from 'page/UnprovisionedSetup';
-import { WelcomePage } from 'page/WelcomePage';
 
-import { CheckList } from './CheckList';
-import { ChooseCheckGroup } from './ChooseCheckGroup';
-import { getRoute } from './Routing.utils';
-import { SceneRedirecter } from './SceneRedirecter';
+import { CheckList } from '../components/CheckList';
+import { ChooseCheckGroup } from '../components/ChooseCheckGroup';
+import { SceneRedirecter } from '../components/SceneRedirecter';
```

Comment:
Imports can be simplified
```suggestion
import { CheckList } from 'components/CheckList';
import { ChooseCheckGroup } from 'components/ChooseCheckGroup';
import { SceneRedirecter } from 'components/SceneRedirecter';
```